deploy/jjb-deploy.yaml

```diff
@@ -22,7 +22,7 @@ objects:
         spec:
           containers:
             - name: jjb
-              image: 172.30.254.79:5000/ceph-csi/jjb:latest
+              image: image-registry.openshift-image-registry.svc:5000/ceph-csi/jjb:latest
```

Comment:
Is this port constant ?

Reply:
yes, I've copied it from the ImageStream from the OCP Console.